core/deployment/src/main/java/io/quarkus/deployment/builditem/DevServicesResultBuildItem.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Configuration property that is the result of start dev services.
+ *
+ * Used to start and configure dev services in native mode.
+ */
+public final class DevServicesResultBuildItem extends MultiBuildItem {
```

Comment:
If we have a dev service depending on another one, how do you get the right key/value? You need to iterate over the list, but because the second one produces also a DevServicesResultBuildItem, it creates a cycle.

Reply:
I renamed the build item to make it clear it is only for native.